src/Data/Either.purs

```diff
@@ -111,7 +111,7 @@ instance altEither :: Alt (Either e) where
 -- |
 -- | ``` purescript
 -- | Left x >>= f = Left x
--- | Right x >>= f = f x
+-- | Right x >>= f = Right (f x)
```

Comment:
This isn't correct. What is the output of `Right x >>= (\_ -> Left 1)`? It's not `Right (f x)`, but `f x`.